crates/e2e/src/xts.rs

```diff
@@ -398,20 +410,20 @@ where
     /// contains all events that are associated with this transaction.
     pub async fn call(
         &self,
-        contract: sp_runtime::MultiAddress<E::AccountId, ()>,
+        contract: MultiAddress<E::AccountId, ()>,
         value: E::Balance,
         gas_limit: Weight,
         storage_deposit_limit: Option<E::Balance>,
         data: Vec<u8>,
         signer: &Signer<C>,
     ) -> ExtrinsicEvents<C> {
-        let call = subxt::tx::StaticTxPayload::new(
+        let call = subxt::tx::Payload::new_static(
```

Comment:
You could also use `Payload::new()` to use the documented interface (though downside is it would allocate since it doesn't specialise on static strings)